entrypoint.sh

```diff
@@ -15,12 +16,7 @@ sh -c "git config --global core.askPass /cred-helper.sh"
 sh -c "git config --global credential.helper cache"
 sh -c "git remote add mirror $*"
 sh -c "echo pushing to $branch branch at $(git remote get-url --push mirror)"
-sh -c "git push mirror $branch"
-
-if [[ -z "${GITLAB_PASSWORD}" ]]; then
-  echo "Empty or non-existent GitLab password."
-  exit 1
-fi
+git push mirror $branch
```

Comment:
Why not run this command in its own shell?

Reply:
I wanted to see if this masks the error and forgot to change it back